xlsynth-driver/src/main.rs

```diff
@@ -66,149 +67,158 @@ trait AppExt {
             .arg(
-                Arg::with_name("flop_inputs")
+                Arg::new("flop_inputs")
                     .long("flop_inputs")
                     .value_name("BOOL")
-                    .takes_value(true)
-                    .possible_values(&["true", "false"])
-                    .min_values(0)
+                    .action(ArgAction::Set)
+                    .value_parser(["true", "false"])
+                    .num_args(0)
                     .help("Flop input ports"),
             )
             .arg(
-                Arg::with_name("flop_outputs")
+                Arg::new("flop_outputs")
                     .long("flop_outputs")
                     .value_name("BOOL")
-                    .takes_value(true)
-                    .possible_values(&["true", "false"])
-                    .min_values(0)
+                    .action(ArgAction::Set)
+                    .value_parser(["true", "false"])
+                    .num_args(0)
                     .help("Flop output ports"),
             )
             .arg(
-                Arg::with_name("add_idle_output")
+                Arg::new("add_idle_output")
                     .long("add_idle_output")
                     .value_name("BOOL")
-                    .takes_value(true)
-                    .possible_values(&["true", "false"])
-                    .min_values(0)
+                    .action(ArgAction::Set)
+                    .value_parser(["true", "false"])
+                    .num_args(0)
                     .help("Add an idle output port"),
             )
             .arg(
-                Arg::with_name("module_name")
+                Arg::new("module_name")
                     .long("module_name")
                     .value_name("MODULE_NAME")
                     .help("Name of the generated module"),
             )
             .arg(
-                Arg::with_name("array_index_bounds_checking")
+                Arg::new("array_index_bounds_checking")
                     .long("array_index_bounds_checking")
                     .value_name("BOOL")
-                    .takes_value(true)
-                    .possible_values(&["true", "false"])
-                    .min_values(0)
+                    .action(ArgAction::Set)
+                    .value_parser(["true", "false"])
+                    .num_args(0)
                     .help("Array index bounds checking"),
             )
             .arg(
-                Arg::with_name("separate_lines")
+                Arg::new("separate_lines")
                     .long("separate_lines")
                     .value_name("BOOL")
-                    .takes_value(true)
-                    .possible_values(&["true", "false"])
-                    .min_values(0)
+                    .action(ArgAction::Set)
+                    .value_parser(["true", "false"])
+                    .num_args(0)
                     .help("Separate lines in generated code"),
             )
+            .arg(
```

Comment:
@cdleary PTAL, I had to add declaration for this flag (not sure how it worked without this until now?)